grails-forge-core/src/main/java/org/grails/forge/feature/github/workflows/plain/templates/plainGithubWorkflow.rocker.raw

```diff
@@ -12,23 +12,24 @@ on: [push, pull_request]
 jobs:
   build:
     runs-on: ubuntu-latest
+    permissions:
+      checks: write
+      contents: read
     steps:
 @javaSetup.template(jdkVersion)
+      - name: Setup Gradle
+        uses: gradle/actions/setup-gradle@@v3
       - name: Run Tests
         if: github.event_name == 'pull_request' || github.event_name == 'workflow_dispatch'
         id: tests
-        uses: gradle/gradle-build-action@@v2
-        with:
-          arguments: check
+        run: ./gradlew check
       - name: Run Build
         if: github.event_name == 'push'
         id: build
-        uses: gradle/gradle-build-action@@v2
-        with:
-          arguments: build
+        run: ./gradlew build
       - name: Publish Test Report
-        if: steps.build.outcome == 'failure' || steps.tests.outcome == 'failure'
+        if: failure()
         uses: scacap/action-surefire-report@@v1
         with:
           github_token: ${{ secrets.GITHUB_TOKEN }}
-          report_paths: '**/build/test-results/test/TEST-*.xml'
\ No newline at end of file
+          report_paths: '**/build/test-results/**/TEST-*.xml'
```

Comment:
Check for files in the `build/test-results/integrationTest` directory as well

Reply:
I think this might be removed as you can view the test results with Gradle build scans.

Reply:
That's a good point. In that case, we should change the template to automatically publish the Gradle build scan as described in this [Gradle documentation](https://docs.gradle.org/current/userguide/github-actions.html#sec:configure_github_actions), but with some comments to let Grails users know what's going on.

```
- name: Setup Gradle
  uses: gradle/actions/setup-gradle@v3
  with:
    # Publish a Gradle build scan so you can see all the build logs,
    # a complete task timeline, test outputs, and the resolved dependencies of your build.
    build-scan-publish: true
    build-scan-terms-of-use-url: "https://gradle.com/terms-of-service"
    build-scan-terms-of-use-agree: "yes"
```
Do you agree? If so, I'll update this pull request.

Reply:
On second thought, I do not think it is a good idea to agree to the term-of-service by default on behalf of the user.
I think it is better to remove, or comment out the build scan params, so the users do an active choice of enabling it.

Reply:
@matrei I agree. I think having the build scan params commented out is a good default.

In this case, I think leaving the `scacap/action-surefire-report@v1` action is a good default so users can easily see their test results.